### PR TITLE
KFSPTS-23942 upgrade version of financials to 6/17/21

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
retest this please

File: src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorAchDataProcessingIntoKfsServiceImpl.java
Comment (cah292) [inline src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorAchDataProcessingIntoKfsServiceImpl.java:105]: If it's no longer necessary for this method to return a boolean, it would be a good idea to adjust its name accordingly. At the moment its name implies that it returns a boolean value.

**Review by cah292 — CHANGES_REQUESTED** [review]
The revised changes look good overall, though I found a few more items that might need cleanup.

File: pom.xml
Comment (db483) [reply pom.xml:2338]: I added the exclusions. I am looking into whether we should update the version for jackson-databind.

**Comment by cah292** [comment]
retest this please

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
After running the upgrade SQL against utest, only 2 tests are failing. They both fail calling into sciquest. I also get an error in KFS-DEV when trying to get into eshop. I think this is related to the sciequest end point not being available. I will retry thus later.

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
The unit tests have now passed. I will refresh utest.

**Review by cah292 — COMMENTED** [review]
Revised changes look good. Unless follow-up updates for the suspension category setup are needed for this PR, this should be ready to merge once we get approval to do so.

**Pull request closed without merging**
